rpc/src/history_rpc.rs

```diff
@@ -170,6 +170,9 @@ impl HistoryNetworkApiServer for HistoryNetworkApi {
     }
 
     /// Send an OFFER request with given ContentKey, to the designated peer and wait for a response.
+    /// If the content value is provided, a "populated" offer is used, which will not store the
```

Comment:
Should we add this to other `*_rpc.rs` files?

Related to this, maybe state shouldn't supported "unpopulated" version.

Reply:
Maybe this is true for beacon network as well?